build/README.Debian.tmpl

```diff
@@ -3,5 +3,5 @@
 
 @@PRODUCT_BASE_CAP@@ Server distribution for Debian
 
-After installing this package, please visit http://hostname:8091
+After installing this package, please visit http://localhost:4985/_admin/
```

Comment:
SG's admin UI is also deprecated, shouldn't recommend it here.

Reply:
This change has been removed.